core/src/components/checkbox/checkbox.ionic.scss

```diff
@@ -8,7 +8,8 @@
   --border-width: #{globals.$ionic-border-size-025};
   --border-style: #{globals.$ionic-border-style-solid};
   --border-color: #{globals.$ionic-color-neutral-800};
-  --checkmark-width: 3;
+  --checkmark-width: #{globals.$ionic-scale-400};
```

Comment:
I think as these variables are in px they won't work for the --checkmark-width. 

Maybe we can add something like [this](https://css-tricks.com/snippets/sass/strip-unit-function/)?

Reply:
I moved it to apply to the svg `width` instead of the path here: https://github.com/ionic-team/ionic-framework/pull/29623/files#diff-796ffc72b6171910c25d5aad085bc245095c38f104d55e253e06282702205066R154

Before:

```scss
.checkbox-icon path {
  fill: none;
  stroke: var(--checkmark-color);
  stroke-width: var(--checkmark-width);

  opacity: 0;
}
```

After:

```scss
.checkbox-icon {
  width: var(--checkmark-width);
  height: var(--checkmark-height);
}

.checkbox-icon path {
  fill: var(--checkmark-color);

  opacity: 0;
}
```

Since the Phosphor Icon doesn't use `stroke` it needs to be applied to the svg itself. This is different than the `ios` and `md` themes, but I think this makes more sense. What do you think?

Reply:
That makes sense yes! 

But it still is not related to the `px` issue, right? Or am I missing something? 🤔 

Its causing this change in snapshot here: https://github.com/ionic-team/ionic-framework/pull/29623#discussion_r1662805252